### ISSUE TITLE
Singleuser: Operations with workspaces don't work

### ISSUE THREAD

**Issue by sleshchenko** [eclipse-che/che#19038]
### Describe the bug
Singleuser: Create workspace from template/sample flow does not work fully/

### Che version

 - [ ] latest
 - [ ] nightly
 - [ ] other: please specify

### Steps to reproduce
1. chectl server:deploy --platform=minikube --installer=helm
2. try to create a workspace from get started or create custom workspace
3. Workspace is created but not started. See the errors:
![Screenshot_20210211_113158](https://user-images.githubusercontent.com/5887312/107620162-f3529a00-6c5c-11eb-82e0-5a36c14178f0.png)

![Screenshot_20210211_113141](https://user-images.githubusercontent.com/5887312/107620158-f2ba0380-6c5c-11eb-8cda-2f91a0e4f08f.png)

If you try to start a workspace directly, you'll get the same error:
![Screenshot_20210211_124059](https://user-images.githubusercontent.com/5887312/107626512-93f98780-6c66-11eb-99a1-dddde1f4043f.png)

### Expected behavior
Workspace is created and started.

### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [ ] Openshift (include output of `oc version`)
   - [ ] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [ ] chectl
     * provide a full command that was used to deploy Eclipse Che (including the output)
     * provide an output of `chectl version` command
   - [ ] OperatorHub
   - [ ] I don't know

### Environment
   - [ ] my computer
       - [ ] Windows
       - [ ] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [ ] other: please specify

**Comment by frankae** [comment]
Hi all,
 have the same issue on docker-desktop 
Windows10 Pro Version 20H2, 
Build 19042.804, 
Windows Feature Experience Pack 120.2212.551.0
Docker Engine v20.10.2
Kubernetes v1.19.3
on an PC
Processor AMD Ryzen 7 2700 
Installed RAM 64GB

Issue is valid in WSL2 and Hyper-V VM mode. 

Installation was made with chectl, tested for single and multiuser. Both didn't work.
chectl server:deploy -p docker-desktop -i helm -m  (that was fine)
chectl server:start (seems to be fine)

Creation of workspaces stopped in Step 2.

**Comment by sleshchenko** [comment]
@frankae please create another issue when you face the same issue on docker-desktop on latest Che, where https://github.com/eclipse/che-dashboard/pull/174 fix is present.

**Comment by t1m0thyj** [comment]
Hello, I'm seeing this issue again in latest version of Che.

As in the original issue, I deployed with `chectl server:deploy --platform minikube --installer helm`

I've tried creating multiple types of workspaces, from a sample or from a devfile, and always get the same error:
![image](https://user-images.githubusercontent.com/22344007/116088899-4f2f9b00-a670-11eb-8d40-725b40ff6520.png)
The error message is the same as the one reported in this issue. Should I create a new issue since this one is closed?

* Version: 7.29.0
* Username: che
* Browser Name: chrome
* Browser Version: 89.0.4389
* Browser OS: Windows 10
